web/app/themes/dp-timber-theme/components/molecule/Card/card.scss

```diff
@@ -0,0 +1,3 @@
+.card-container {
+    max-width: 600px;
```

Comment:
why do we need this on the molecule level?

Reply:
No I only this for Storybook, should I rewrite this CSS in storybook override?

Reply:
I've created a card.storybook.scss inside card folder to use scss only for storybook